Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/comments/CommentsDialog.java

```diff
@@ -269,9 +269,13 @@ private JPanel createPanel() {
 			JTextArea currentTextArea = getSelectedTextArea();
 			AnnotationAdapterWrapper aaw =
 				(AnnotationAdapterWrapper) annotationsComboBox.getSelectedItem();
-			currentTextArea.insert(aaw.getPrototypeString(),
-				currentTextArea.getCaretPosition());
-			currentTextArea.setCaretPosition(currentTextArea.getCaretPosition() - 1);
+			if (currentTextArea.getSelectedText() != null && !currentTextArea.getSelectedText().isEmpty()) {
```

Comment:
- Please extract the value of `getSelectedText()` into a variable.
- Instead of the `null` check and the empty check, use one call to `StringUtils.isBlank()`